.rubocop.yml

```diff
@@ -5,7 +5,7 @@ require:
 AllCops:
   DisplayCopNames: true
   DisplayStyleGuide: true
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 2.6
```

Comment:
Updated this based on the minimum Ruby version in [that matrix](https://github.com/lostisland/faraday-net_http/blob/main/.github/workflows/ci.yml#L34)